test/validators.js

```diff
@@ -113,6 +118,60 @@ describe('Validators', () => {
         'multiple..dots@gmail.com',
         'wrong()[]",:;<>@@gmail.com',
         '"wrong()[]",:;<>@@gmail.com',
+        '""@invalid.com',
+        '-@gmail.com',
+        '+@gmail.com',
+        '+@test.com',
+        '-abc@mail.com',
+        '+abc@mail.com',
+        'abc-@mail.com',
+        'abc..def@mail.com',
+        '_abc_df@domain.com',
+        '!abc@mail.com',
+        'abc!@email.com',
+        '#abc@mail.com',
+        'abc#@email.com',
+        '$abc@mail.com',
+        'abc$@email.com',
+        '%abc@mail.com',
+        'abc%@email.com',
+        '&abc@mail.com',
+        'abc&@email.com',
+        '\'abc@mail.com',
+        'abc\'@email.com',
+        '*abc@mail.com',
+        'abc*@email.com',
+        '+abc@mail.com',
+        'abc+@email.com',
+        '-abc@mail.com',
+        'abc-@email.com',
+        '/abc@mail.com',
+        'abc/@email.com',
+        '=abc@mail.com',
+        'abc=@email.com',
+        '?abc@mail.com',
+        'abc?@email.com',
+        '^abc@mail.com',
+        'abc^@email.com',
+        '_abc@mail.com',
+        'abc_@email.com',
+        '`abc@mail.com',
+        'abc`@email.com',
+        'ab++c@email.com',
+        '{abc@mail.com',
+        'abc{@email.com',
+        '|abc@mail.com',
+        'abc|@email.com',
+        '..abc@mail.com',
+        'ab--cc@mail.com',
```

Comment:
why isn't this a valid email?

Reply:
It is a valid email. The issue is that it seemed that there was an article that I read on having special characters that don't follow @ other as invalid email. But, I realized that it only happens in the case of `dot`. I will take a look at these changes once again.